var/spack/repos/builtin/packages/zziplib/package.py

```diff
@@ -6,7 +6,7 @@
 from spack import *
 
 
-class Zziplib(AutotoolsPackage):
+class Zziplib(AutotoolsPackage, CMakePackage):
```

Comment:
OK @alalazo, so I switch back to AutotoolsPackage only as you recommend ; may be in the future it will switch to CMakePackage only when 0.13.69 will be considered deprecated.